doc/website/src/_static/custom.css

```
@@ -0,0 +1,105 @@
+/* GenLayer GenVM SDK — Custom Styles */
+
+/* ── Font ── */
+@font-face {
+  font-family: 'Switzer';
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Switzer" (font-family-name-quotes)

(font-family-name-quotes)